ikenna-portfolio/portfolio/src/main/webapp/test.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
```

Comment:
Maybe you could consider renaming this to something more descriptive than test.html. Not blocking for this pull request though since the demo is tomorrow.

Reply:
will do, will probably become index.html unless I find out that it's not a good solution

Reply:
done